### ISSUE TITLE
missing xrefs for Microsoft.ML links

### ISSUE THREAD

**Issue by nschonni** [dotnet/docs#11219]
Noticed this on the OpenBuild logs. Not sure if i'm getting this because I was the last one to touch the doc.
You can see on the page that they're not rendering anymore as well

**Log code info**
(This is auto filled, no need to modify)

| Severity   | Type   | Code   | Category   | File   | Message   |  Source  | Line   | Time   |
|------------|--------|--------|------------|--------|-----------|----------|--------|--------|
| Warning | Unspecified | UidNotFound | Markdown | docs/machine-learning/resources/transforms.md | 19 invalid cross reference(s) "<xref:Microsoft.ML.Transforms.OnnxTransform>", "<xref:Microsoft.ML.Transforms.TensorFlowTransform>", "<xref:Microsoft.ML.Transforms.Text.CustomStopWordsRemovingTransform>", "<xref:Microsoft.ML.ImageAnalytics.ImageGrayscaleTransform>", "<xref:Microsoft.ML.ImageAnalytics.ImageLoaderTransform>", "<xref:Microsoft.ML.ImageAnalytics.ImagePixelExtractorTransform>", "<xref:Microsoft.ML.ImageAnalytics.ImageResizerTransform>", "<xref:Microsoft.ML.Transforms.Projections.PcaTransform>", "<xref:Microsoft.ML.Transforms.LabelIndicatorTransform>", "<xref:Microsoft.ML.Transforms.Normalizers.MeanVarDblAggregator>", "<xref:Microsoft.ML.Transforms.Normalizers.MeanVarSngAggregator>", "<xref:Microsoft.ML.Transforms.Normalizers.MinMaxDblAggregator>", "<xref:Microsoft.ML.Transforms.Normalizers.NormalizeTransform>", "<xref:Microsoft.ML.Transforms.BootstrapSamplingTransformer>", "<xref:Microsoft.ML.Transforms.SkipTakeFilter>", "<xref:Microsoft.ML.Transforms.OptionalColumnTransform>", "<xref:Microsoft.ML.Transforms.RangeFilter>", "<xref:Microsoft.ML.Transforms.GenerateNumberTransform>", "<xref:Microsoft.ML.Transforms.RandomNumberGenerator>". | BuildCore.Build Document.LinkPhaseHandlerWithIncremental.Apply Templates | | 03/08/2019 07:10:15 |

**Link to report**
(This is auto filled, no need to modify)

[report url](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C8%5Ca3bda507-3390-de91-8f7f-26f90f4e5fc8%5CPullRequest%5C201903080706583772-11213%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JWUJSLD9LtP3yvB970l8jsYzLEtcN%2B5r%2B38uM0%2BIwIU%3D&st=2019-03-08T07%3A06%3A06Z&se=2019-04-08T07%3A11%3A06Z&sp=r)

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a4185570-21cc-dcdb-62fd-d05bceb62f81
* Version Independent ID: 4f0fe52a-a771-d044-593d-2bb65df15e0d
* Content: [Machine learning data transforms - ML.NET](https://docs.microsoft.com/en-ca/dotnet/machine-learning/resources/transforms)
* Content Source: [docs/machine-learning/resources/transforms.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/resources/transforms.md)
* Product: **dotnet-ml**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

**Comment by BillWagner** [comment]
I don't think you caused this @nschonni 

Ping @JRAlexander I think you and @luisquintanilla can fix this one.

**Comment by JRAlexander** [comment]
It's on the list.

**Comment by nschonni** [comment]
Yeah, I didn't think it was my changes, just more that I touched the file, so the links started to get re-validated and failed :wink:

**Comment by natke** [comment]
Fixed by #11699

**Comment by nschonni** [comment]
Thanks @natke it looks like that PR still has warnings one some
```
docs/machine-learning/resources/tasks.md | 12  invalid cross reference(s)  "Microsoft.ML.Trainers.Online.AveragedPerceptronTrainer",  "Microsoft.ML.FactorizationMachine.FieldAwareFactorizationMachineTrainer",  "Microsoft.ML.Trainers.Online.LinearSvmTrainer",  "Microsoft.ML.Learners.LogisticRegression",  "Microsoft.ML.Trainers.StochasticGradientDescentClassificationTrainer",  "Microsoft.ML.Trainers.SymSgd.SymSgdClassificationTrainer",  "Microsoft.ML.Learners.MetaMulticlassTrainer`2",  "Microsoft.ML.Trainers.Online.OnlineGradientDescentTrainer",  "Microsoft.ML.Trainers.FastTree.SingleTrainer",  "Microsoft.ML.LightGBM.SingleTrainer",  "Microsoft.ML.Trainers.KMeans.KMeansPlusPlusTrainer",  "Microsoft.ML.Trainers.PCA.RandomizedPcaTrainer".
```

**Comment by natke** [comment]
Ah yes. Thank you. We will be updating that doc soon too

**Comment by natke** [comment]
Remainder of errors fixed by #11820
